web/lib/firebase/seen-contracts.ts

```diff
@@ -0,0 +1,22 @@
+import { mapValues } from 'lodash'
```

Comment:
This is good, although I would have just as soon removed it all, b/c I'm not sure we're using the seen contracts in any way today.  (It used to track contracts seen in the old feed.)

Actually, maybe we should still axe this and the visibility observer within the feed items, just to simplify.

Reply:
If you're cool with it of course I would love to axe it!